lib/internal/errors.js

```diff
@@ -148,6 +148,7 @@ E('ERR_SOCKET_BAD_TYPE',
 E('ERR_SOCKET_CANNOT_SEND', 'Unable to send data');
 E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536');
 E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
+E('ERR_INVALID_FD', 'fd must be positive integer');
```

Comment:
Yeah, at this point in time it's a best effort while we get all these done. There are many separate PRs and things are going to get out of order. We need to just stay on top of making sure they get organized *eventually*.